src/Tools/AzureBlobStorage/CollectionRuleDefaultsOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options
+{
+    internal class CollectionRuleDefaultsOptions
```

Comment:
What requires these two classes? Egress should be independent of collection rules.

Reply:
This was required by `RootOptions` which is pulled in from `LoggingExtension`. I pulled that out and made a new one with just the few extensions that are needed and was able to remove this and pair down usage of all of this.